packages/styles/scss/components/accordion/_accordion.scss

```
@@ -285,19 +302,75 @@ $content-padding: 0 0 0 $spacing-05 !default;
   visibility: hidden;
 }
 
-// flush
-.#{$prefix}--accordion--flush .#{$prefix}--accordion__title {
-  margin-inline-start: 0;
+//flush
+
+// .#{$prefix}--accordion--flush .#{$prefix}--accordion__item {
+//   position: relative;
+//   border-color: transparent;
+
+//   &:last-child {
+//     border-color: transparent;
+//   }
+
+// }
+
+.#{$prefix}--accordion--flush .#{$prefix}--accordion__item {
+  position: relative;
+  border-color: transparent;
 }
 
-.#{$prefix}--accordion--flush .#{$prefix}--accordion__arrow {
-  margin-inline-end: 0;
+.#{$prefix}--accordion--flush .#{$prefix}--accordion__item:last-child {
+  border-color: transparent;
 }
 
-.#{$prefix}--accordion--flush .#{$prefix}--accordion__content {
-  padding-inline-start: 0;
+.#{$prefix}--accordion--flush .#{$prefix}--accordion__item::after {
+  position: absolute;
+  display: block;
+  block-size: 100%;
+  border-block-start: 1px solid $border-subtle;
+  content: '';
+  inline-size: calc(100% - $spacing-05 - $spacing-05);
+  inset-block-start: -1px;
+  margin-inline-start: $spacing-05;
```

Same as above, can we add a transition so the hover styles match the background color change?

```suggestion
  margin-inline-start: $spacing-05;
  transition: border-color motion(standard, productive) $duration-fast-02;
```